### ISSUE TITLE
Manually Adding Actors Issue

### ISSUE THREAD

**Issue by AlphaAlta** [PAhelper/PhoenixAdult.bundle#1275]
The manually adding actors feature added in #155 is giving me some issues.
Basically I can add the actor, but then her first name shows up as actor as well.

Example: add - actor Kira Adams

That will successfully add a new actor named "Actor Kira Adams". 

And if you remove the word actor from the example, then no actor gets added at all.

**Comment by DirtyRacer1337** [comment]
Sjould be somelthing like `Add Kira Adams In Title At Studio`

**Comment by MrDemous** [comment]
> Should be something like `Add Kira Adams In Title At Studio`

Sorry for hijacking the ticket but I didn't want to start a new one just to ask a question.

Can this be used to add genres and date as well? And if so what syntax should I use?

**Comment by DirtyRacer1337** [comment]
> > Should be something like `Add Kira Adams In Title At Studio`
> 
> Sorry for hijacking the ticket but I didn't want to start a new one just to ask a question.
> 
> Can this be used to add genres and date as well? And if so what syntax should I use?

nah, but you can do it in plex

**Comment by MrDemous** [comment]
Can it be added in the future? because it'd be a great addition for videos download from Pornhub. I can basically add all needed info from filenames if the file is named properly.

**Comment by DirtyRacer1337** [comment]
> Can it be added in the future? because it'd be a great addition for videos download from Pornhub. I can basically add all needed info from filenames if the file is named properly.

Added in b72d9903a6d0aceeea50757b6f8d0d1311d804dc

Should work with `Add Actor 1, Actor 2 In Title At Studio With genre 1, genre 2`

**Comment by MrDemous** [comment]
@DirtyRacer1337 You're a genius. Thank you so much.

**Comment by MrDemous** [comment]
Got another question/request. Is there anyway that we can distinct between in/at/with needed to add actors, studios etc... And the `in` `at` `with` in some scene titles?

Maybe change the syntax needed from just `in` to `(in)` so that PA can differentiate between the two?

**Comment by DirtyRacer1337** [comment]
> Got another question/request. Is there anyway that we can distinct between in/at/with needed to add actors, studios etc... And the `in` `at` `with` in some scene titles?
> 
> Maybe change the syntax needed from just `in` to `(in)` so that PA can differentiate between the two?

Send some examples, I'll think about improving

**Comment by MrDemous** [comment]
This scene for example `Sasha leaves you in ruins`

If I use `Add Sasha Foxxx in Sasha leaves you in ruins at MHBHJ` PA will add the scene title as `Ruins` and the actor as `Sasha Foxxx in Sasha leaves you`

What I suggest is just adding the words needed for PA to detect actors/titles/genres in bracket so that it doesn't conflict with scene titles that has "in", "with" or "at" in the title name.

Basically changing the syntax from `Add Sasha Foxxx in Sasha leaves you in ruins at MHBHJ` to `Add Sasha Foxxx (in) Sasha leaves you in ruins (at) MHBHJ`

**Comment by DirtyRacer1337** [comment]
@MrDemous try with c6d0376eaaab3d3434c5126da2093cc47c35644f, hope it's not broke others

**Comment by MrDemous** [comment]
The change worked with the scene I mentioned before `Sasha Leaves You in Ruins`

But I tried another scene `Unexpected Afternoon Sex With My Excited Love in Fuerteventura!`

I used `Add Danika Mori in Unexpected Afternoon Sex With My Excited Love in Fuerteventura! at Pornhub` and this adds the actress fine but doesn't add the studio and gives a title of `Unexpected Afternoon Sex`

**Comment by DirtyRacer1337** [comment]
@MrDemous Try a95a4540c6daadb80d6dd7f83666dc3303a007db

Now you can put them in double quotes

**Comment by MrDemous** [comment]
Nope, not working. Still gives the same result. I tried both single and double quotes

**Comment by DirtyRacer1337** [comment]
With this 4228838f8d415f4face69369e520d140b5532773?

**Comment by MrDemous** [comment]
Yeah, I was just about to mention that.

**Comment by DirtyRacer1337** [comment]
Alright try on latest, you can add empty `at` `with` like `Add Danika Mori in Unexpected Afternoon Sex With My Excited Love in Fuerteventura! at Pornhub with`

**Comment by MrDemous** [comment]
Works great if I use `Add Danika Mori in Unexpected Afternoon Sex With My Excited Love in Fuerteventura! at Pornhub with` but if I remove the last `with` it breaks again.

**Comment by DirtyRacer1337** [comment]
because `with` in title

**Comment by MrDemous** [comment]
So any scene that has `with` in its title, I have to add with at the end for it to work properly?

**Comment by DirtyRacer1337** [comment]
yes

**Comment by MrDemous** [comment]
Okay, thanks a lot for your help and amazing work @DirtyRacer1337

**Comment by MrDemous** [comment]
Just wanted to say thank you again @DirtyRacer1337 for your amazing work. I just used this to match around 1100 videos ;)

**Comment by DirtyRacer1337** [comment]
> Just wanted to say thank you again @DirtyRacer1337 for your amazing work. I just used this to match around 1100 videos ;)

No problem 👍
